### ISSUE TITLE
Is the repo still alive? 

### ISSUE THREAD

**Issue by vvscode** [h5bp/Front-end-Developer-Interview-Questions#409]
I see talks from folks. But no merged PR for last 2 months.

**Comment by SenHeng** [comment]
Good question. Quick browser of the h5bp project page and it seems like activity on the other projects seem to have cooled off too.

**Comment by bendtherules** [comment]
Is there any active fork of this repo? Somewhere people can contribute in the meantime?

**Comment by vvscode** [comment]
@AD7six,@alexgibson,@alrra,@appleboy,@arthurvr,@beverloo,@bgrins,@brianblakely,@carsonmcdonald,@chriscoyier,@ChrisMcKee,@connor,@darcyclarke,@drublic,@h5bp-bot,@jonathantneal,@KushalP,@mathiasbynens,@mischah,@mklabs,@necolas,@ngryman,@niftylettuce,@nimbupani,@paulirish,@pifantastic,@Raynos,@richgilbank,@roblarsen,@shichuan, @sindresorhus,@tagawa,@tbranyen,@WellingGuzman,@Wilto,@x1ddos,@XhmikosR,@xonecas,@zenorocha

Any answer? Or maybe you can add someone who can manage the repo?

**Comment by ChrisMcKee** [comment]
IIS Repo is still alive which tbh is all I maintain. Just little to add at the moment to that as .net goes multiplatform but the repo's very much for the server.

**Comment by vvscode** [comment]
@ChrisMcKee  Looks like I don't understand your answer ))

**Comment by ChrisMcKee** [comment]
@vvscode you added me to it as I'm in the h5bp group but I only maintain 1 repo in it. 
I had no idea this repo existed; I don't have any ability to manage it.
If anyone wants to volunteer to maintain this repo, post below and I'm sure one of the 'owners' can add that person as the maintainer. 
@paulirish @alrra @drublic

**Comment by cezaraugusto** [comment]
hey all I'd like to volunteer to maintain this. This has a lot of value for our community and the web evolves fast, so questions eventually need to be added/updated and there's so much value to add from contributors that we're missing now. 

ATM we have PRs dated from 2015 which makes the repo seems stale -- which leads contributors to stop proposing changes, which is sad. I'd be very happy to do the update work and make this even better.

**Comment by roblarsen** [comment]
> hey all I'd like to volunteer to maintain this. This has a lot of value for our community and the web evolves fast, so questions eventually need to be added/updated and there's so much value to add from contributors that we're missing now.

I just invited you to the team 👍  

Help is still wanted, so if anyone else would like to help out around here let me know. I'll also watch the repo for a little while and help out where applicable.  I'm going to be a little busy for the next few weeks, but I agree that this is a good resource, so I'll pitch in where I can to get it resuscitated.

**Comment by Snowbell92** [comment]
I'd like to volunteer too. Fair warning, I'm still a junior dev, so I'll probably need a lot of guidance.

**Comment by vvscode** [comment]
@roblarsen I would like to help too. So fill free to add me into the repo

**Comment by bendtherules** [comment]
@roblarsen Hi Rob, I would also like to help with adding more questions. If the maintainer vacancy is filled up, I can add those via PR.

A go-to maintainer would really help - someone with maybe a little more time and responsibility to make sure that things don't pile up again.

**Comment by roblarsen** [comment]
![its-alive-gif-13](https://user-images.githubusercontent.com/361421/30651061-96dbbcfc-9df2-11e7-9934-df5b35fc15f9.gif)

**Comment by roblarsen** [comment]
@bendtherules @Snowbell92 I'll email the both of you about helping out.

**Comment by SenHeng** [comment]
@roblarsen 

I'd like to help too 🖖
